Server/service/networkService.js

```
@@ -100,7 +124,7 @@ class NetworkService {
 		} catch (error) {
 			const endTime = Date.now();
 			error.responseTime = endTime - startTime;
-			error.service === undefined ? (error.service = SERVICE_NAME) : null;
+			error.service === undefined ? (error.service = this.SERVICE_NAME) : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)